tests/src/JIT/HardwareIntrinsics/X86/Sse2/TestTableSse2.cs

```diff
@@ -224,7 +224,7 @@ public bool CheckResult(CheckMethod<T> check)
         public bool CheckResult(CheckMethodSpan<T> check)
         {
             bool result = true;
-            for (int i = 0; i < inArray1.Length; i++)
+            for (int i = 0; i < inArray1.Length; i += ElementCount)
```

Comment:
Why this change?

Reply:
This method was not used before and was created specifically for `Span<T>` arguments. It needs to increment by element count of `Vector<T>`